lib/rspec-console.rb

```diff
@@ -45,12 +40,21 @@ def red
           Otherwise, code relading does not work.
         MSG
       else
-        ActionDispatch::Reloader.cleanup!
-        ActionDispatch::Reloader.prepare!
+        Rails.application.reloader.reload!
       end
     end
   end
 
   # Reloading FactoryGirl if necessary
   before_run { FactoryGirl.reload if defined?(FactoryGirl) }
+
+  # This is needed to avoid problem explained at
+  # https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rails-preloaders-and-rspec
+  before_run { FactoryBot.reload if defined?(FactoryBot) }
+
+  # Clear Faker gem unique
+  before_run { Faker::Name.unique.clear }
```

Comment:
Wondering if it would be better to use `Faker::UniqueGenerator.clear` to clear all values instead of explicitly clearing ones we know about?

Reply:
Yeah you suggestions seems to work, thanks!